src/components/Header/headerUtils.ts

```diff
@@ -1,15 +1,82 @@
+/*eslint no-useless-escape: 0 */
 import Cookies from "js-cookie";
+
 export interface Alert {
   id: string;
   link: string;
   description: string;
   startDate: string;
   endDate: string;
 }
-
 export const alertsApiUrl =
   "https://refinery.nypl.org/api/nypl/ndo/v0.1/content/alerts?filter%5Bscope%5D=all";
 
+/**
+ * Replaces the search string's special characters that need to be encoded
+ * using base64. These characters are "=","/", "\", "?".
+ */
+const encoreEncodeSearchString = (searchString) => {
```

Comment:
Personal preference: I like having unit tests for this kind of stuff (and also stuff like extractPatronName) - as it serves as really good documentation for what is expected and what is output.

Reply:
Sometimes personal preference _is_ a best practice.

I'll add tests for this file but I think we got carried away with testing _just_ the UI that not all utility functions are tested as they should.